tests/plugins/pastefromword/generated/PFW_image.js

```diff
@@ -0,0 +1,39 @@
+/* bender-tags: clipboard,pastefromword */
+/* bender-ckeditor-plugins: pastefromword,ajax,basicstyles,bidi,font,link,toolbar,colorbutton,image */
+/* bender-ckeditor-plugins: list,liststyle,sourcearea,format,justify,table,tableresize,tabletools,indent,indentblock,div,dialog */
```

Comment:
I think many of this plugins could be safely removed. Could we do that?

Reply:
I reduced it, but few of them still remain.